planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -2392,22 +2393,6 @@ void AvoidanceModule::updateData()
   }
 }
 
-std::string getUuidStr(const ObjectData & obj)
```

Comment:
@TakaHoribe 
I've modified this function to reflect the hexadecimal property of `uuid`, and moved them to `avoidance_util` as i think that is the suitable location of this function.